src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -492,4 +493,30 @@ protected function createChildContext(array $parentContext, $attribute/*, string
 
         return $parentContext;
     }
+
+    /**
+     * @param array                      $context
+     * @param AttributeMetadataInterface $attributeMetadata
+     *
+     * @return bool
+     */
```

Comment:
The whole doc block can be removed, as it doesn’t add any value. Or add some description and only remove return phpdoc